drawBot/context/tools/imageObject.py

```diff
@@ -56,17 +57,21 @@ def open(self, path):
         """
         if isinstance(path, AppKit.NSImage):
             im = path
-        elif isinstance(path, str):
-            path = optimizePath(path)
+        else:
```

Comment:
Same here, leave the original elif, extend isinstance with `os.PathLike`, get rid of the new else.